ee/docker-compose.ch.arm64.yml

```diff
@@ -77,7 +78,6 @@ services:
         command: '${CH_WEB_SCRIPT:-./ee/bin/docker-ch-dev-web}'
         ports:
             - '8000:8000'
-            - '8234:8234'
```

Comment:
I was wondering about that as I ran into the same issue... but with this, there's no way to run the full stack in dev mode inside docker... which is what we might want at times.